subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -665,6 +684,11 @@ void ll_rx_dequeue(void)
 	case NODE_RX_TYPE_REPORT:
 #endif /* CONFIG_BT_OBSERVER */
 
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
+	/* fallthrough */
```

Comment:
remove comment

Reply:
I think the "fallthrough" comment might be needed by Coverity.

Reply:
In that case it shall stay, but there should be an action item to add the fallthrough comment to the places where it is missing.